README.md

```diff
@@ -151,6 +153,10 @@ experiment = Experiment.new(name="SDK EXP", description="new experiment test for
 client.create(experiment)
 ```
 
+Note: All timestamps on the new experiment must be in Unix Timestamp (Represents time as the number of seconds since the Unix epoch). 
```

Comment:
Do we require a specific precision (nanosecond, milliseconds, etc.) for the timestamp or we accept multiple ones?

Reply:
we just accept seconds. I'll add that here.